python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -341,19 +341,18 @@ def runDataRef(self, butler):
                           "fitCycleConfig.doReferenceCalibration")
 
         # And make sure that the atmosphere was output properly
-        if (self.config.doAtmosphereOutput and
-                not butler.datasetExists('fgcmAtmosphereParameters', fgcmcycle=self.config.cycleNumber)):
-            raise RuntimeError("Atmosphere parameters are missing for cycle %d." %
-                               (self.config.cycleNumber))
+        if (self.config.doAtmosphereOutput
+                and not butler.datasetExists('fgcmAtmosphereParameters', fgcmcycle=self.config.cycleNumber)):
```

Comment:
Is this indent inconsistent?

Reply:
I am mostly keeping the original indentation. I think the point is that you can either under indent or over indent for the `if` but you can't use 4 spaces because that is visually indistinct from the first line in the block.